docs/central-upgrade.rst

```diff
@@ -99,10 +99,10 @@ Upgrading to Central v2024.2
 
 There are no required special steps related to this upgrade. However, there are some optional changes that you may want to know about or opt into.
 
-1. Configuration no longer requires rebuilding
+1. Changes to .env no longer require rebuilding
```

Comment:
I see that the spellcheck is complaining about ".env". Does it need to be enclosed in backticks? I'm happy to make that change if it'd help.